packages/fdc3-context/schemas/context/nothing.schema.json

```diff
@@ -10,14 +10,19 @@
       "properties": {
         "type": {
           "const": "fdc3.nothing"
+        },
+        "contextType": {
+          "type": "Context Type",
```

Comment:
I don't think that we still need this change now that we're not using the `fdc3.nothing` context?

Reply:
Removed, thanks!